MSAL/src/MSALError_Internal.m

```diff
@@ -72,6 +72,10 @@ MSALErrorCode MSALErrorCodeForOAuthError(NSString *oauthError, MSALErrorCode def
     {
         return MSALErrorInvalidClient;
     }
+    if ([oauthError isEqualToString:@"invalid_scope"])
+    {
+        return MSALErrorInvalidParameter;
```

Comment:
Yes, it's intended, but wanted to discuss it :) Personally I think MSALErrorInvalidParameter is good to keep number of different errors low, but I'm fine with creating a new error too.
MSALErrorInvalidParameter = A required parameter was not provided, or a passed in parameter was invalid. See MSALErrorDescriptionKey for more information.
invalid_scope - The requested scope is invalid, unknown, malformed, or exceeds the scope granted by the resource owner.